spec/models/service_order_spec.rb

```diff
@@ -33,4 +33,25 @@
       end
     end
   end
+
+  it "should return a short_his_name" do
+    short_name = ServiceOrder.new(his_name: "John Smith")
+    expect(short_name.his_name) == "John"
+  end
+
+  it "should not return a short_his_name" do
+    short_name = ServiceOrder.new(his_name: "John Smith")
+    expect(short_name.his_name) == (["John Smith", "Smith"])
```

Comment:
Check the expect syntax here:
https://github.com/rspec/rspec-expectations#built-in-matchers